### ISSUE TITLE
NVDA KEY+F7 is not available in HTML messages of Outlook.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3077]
**Reported by sumandogra on 2013-03-12 06:37**
Outlook 2010
NVDA KEY+F7 is not available in the HTML messages of Outlook.  In long HTML messages of Outlook that has a many links; it becomes difficult to select a link quickly.  
Steps to reproduce:
1.  Open a HTML message in Outlook that has links present in it.
2.  Press NVDA KEY+F7.
3.  Nothing happens.

**Comment by ehollig** [comment]
I am able to use NVDA+F7 in Outlook 2016. Is it possible to do this in Outlook 2010 as well?

**Comment by ehollig** [comment]
Note that the author is referring to using NVDA+F7 after tabbing into the message view of Outlook. In my previous comment, I was opening the message in a new window. In the message view of Outlook 2016, NVDA+F7 does not work probably because of what @jcsteh said in https://github.com/nvaccess/nvda/issues/3077#issuecomment-155305744 that "the message is detected and handled like any other instance of a Microsoft Word document." However, when viewing this message and using NVDA+space, it allows me to view the document in browse mode, allowing for NVDA+F7 to work. Thoughts from others?

**Comment by leonardder** [comment]
@ehollig: I think your last comment would be a good rationale for closing this, especially since it is possible to get NVDA+f7 to work by enabling browse mode.

**Comment by ehollig** [comment]
Alright. Thanks for the feedback. Closing
